api/MediaQueryListListener.json

```diff
@@ -5,10 +5,10 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaQueryListListener",
         "support": {
           "chrome": {
-            "version_added": null
+            "version_added": false
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/MediaQueryListListener documents this as a callback function, the argument to https://drafts.csswg.org/cssom-view/#the-mediaquerylist-interface which Chromium does support.

I'd argue this is a bug in BCD, that callback function shouldn't have a separate entry.